es/LEEME.md

```diff
@@ -0,0 +1,222 @@
+# Aprendiendo Bitcoin desde la Línea de Comandos 2.0.1
+*Por Christopher Allen and Shannon Appelcline*
+
+![alt text](https://www.blockchaincommons.com/images/projects/lbtc-screen.png)
+
+Aprender Bitcoin desde la línea de comandos es un tutorial para trabajar con Bitcoin (y Lightning) que enseña mediante la interacción directa con los propios servidores, como la forma más sólida y segura de comenzar a trabajar con criptomonedas.
+
+> NOTA: Este es un borrador en progreso, por lo que puedo obtener algunos comentarios de los primeros revisores. Aún no está listo para usarse.
+
+Este tutorial asume que usted tiene algunos conocimientos previos mínimos sobre cómo usar la interfaz de línea de comandos. Si no es así, hay muchos tutoriales disponibles y tengo uno para usuarios de Mac en [https://github.com/ChristopherA/intro-mac-command-line.](https://github.com/ChristopherA/intro-mac-command-line)
+
+## Tabla de Contenidos
+
+### PARTE UNO: PREPARÁNDOSE PARA BITCOIN
+
+**Estado:** Finalizado. Actualizado para la versión 0.20.
+
+* [1.0: Introducción a la programación con Bitcoin Core y Lightning](01_0_Introduccion.md)
+    * [Interludio: Introduciendo Bitcoin](01_1_Introduciendo_Bitcoin.md)
+* [2.0: Configuración de un Servidor Privado Virtual](02_0_Configurando_un_Bitcoin-Core_VPS.md)
+  * [2.1: Configuración de un Servidor Privado Virtual de Bitcoin-Core con Bitcoin Standup](02_1_Configurando_un_Bitcoin-Core_VPS_con_StackScript.md)
+  * [2.2: Configuración de una maquina Bitcoin-Core por otros medios](02_2_Configurando_Bitcoin_Core_Otros.md)
+
+### PARTE DOS: USANDO BITCOIN-CLI
+
+**Estado:** Finalizado. Actualizado para la versión 0.20.
+
+* [3.0: Entendiendo Su Configuración Bitcoin](03_0_Entendiendo_Su_Configuracion_Bitcoin.md)
+  * [3.1: Verificando Su Configuración Bitcoin](03_1_Verificando_Su_Configuracion_Bitcoin.md)
+  * [3.2: Conociendo Su Configuración Bitcoin](03_2_Conociendo_Su_Configuracion_Bitcoin.md)
+  * [3.3: Configurando Su Billetera](03_3_Configurando_Su_Billetera.md)
+    * [Interludio: Usando Variables de la Interfaz de Linea de Comandos](03_3__Interludio_Usando_Variables_de_la_Interfaz_de_Linea_de_Comandos.md)
```

Comment:
03_3_Interludio_Usando_Variables_Linea_Comando.md

Reply:
Should I change also the link name?

Reply:
Should I also change the link names (inside [])?

Reply:
Yes!, sorry.

Reply:
It's done. I added some articles to make it not so cryptic and more readable, I hope it's ok.